transformer_engine/common/fused_softmax/scaled_masked_softmax_cuda.cu

```diff
@@ -305,6 +305,13 @@ __global__ void scaled_masked_softmax_warp_forward(
     }
     warp_reduce<acc_t, WARP_BATCH, WARP_SIZE, Max>(max_value);
 
+    // compute scale value to account for full mask
+    acc_t scale_value[WARP_BATCH];
+#pragma unroll
+    for (int i = 0;  i < WARP_BATCH;  ++i) {
+        scale_value[i] = (max_value[i] == -10000.0) ? 0.0 : 1.0;
```

Comment:
`-10000.0` seems like a very specific number, maybe at least do `<=`?

Reply:
This is set previously in line 284 and we would presumably want the exact value.